common.yml

```
@@ -641,6 +641,37 @@ functions:
               --package-file-suffix "" \
               release.msi
 
+  "sign linux packages":
+    - command: shell.exec
+      type: system
+      params:
+        working_dir: src/github.com/mongodb/mongo-tools/
+        silent: true
+        script: |
+          echo "${signing_auth_token_server_4_4}" > ./signing_auth_token
+    - command: shell.exec
+      type: system
+      params:
+        working_dir: src/github.com/mongodb/mongo-tools/
+        silent: true
+        script: |
+          /usr/local/bin/notary-client.py \
```

```suggestion
          set -o errexit
          /usr/local/bin/notary-client.py \
```
Without this, the task won't actually fail if the first notary-client invocation errors, but the second one succeeds

---

Should this also be set in the `sign msi installer` task?

---

It should be set in an shell task unless we have a specific reason to *not* set it :)